app/screens/receive-bitcoin-screen/receive-btc.tsx

```diff
@@ -448,6 +449,12 @@ const ReceiveBtc = () => {
     )
   }
 
+  if (state === PaymentRequestState.Paid) {
+    ReactNativeHapticFeedback.trigger("notificationSuccess", {
+      ignoreAndroidSystemSettings: true,
```

Comment:
why is that necessary?

Reply:
this triggers the haptic feedback when the receive screen is on and that is paid( basically when the tick mark icon comes up).